auth-tokens-filter/src/main/java/com/palantir/tokens/auth2/http/BearerTokenLoggingFilter.java

```diff
@@ -79,6 +79,6 @@ private void setUnverifiedContext(ContainerRequestContext requestContext, String
     }
 
     public static String getRequestPropertyKey(String key) {
-        return "com.palantir.tokens.auth." + key;
+        return "com.palantir.tokens.auth2." + key;
```

Comment:
hmm, this is tricky. on the fence if this should be changed or not. what was your rationale for changing it?